### PR TITLE
Design Teardown - Microverse Project Submission 

### PR DESCRIPTION
**Pull request by umairarshadbutt**
This pull request is for the clone of [Smashing Magazine ](https://www.smashingmagazine.com/)

### CODE REVIEW

**Comment by elmejdki** [comment]
### Pull Request under review by @Zakariae

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
-# Design-Teardown
\ No newline at end of file
+#Design Teardown
```
Comment (elmejdki) [inline README.md]: Add a space after the hash to allow markdown styling.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+                    </article>
+                    <a href="#" class="arrow">→</a>
+                </div>
+               
+            </div>
+           
+        </div>
+
+        <section class="latest">
+            <div class="column4" >
```
Comment (elmejdki) [inline index.html]: Please use article tags here instead of div's for all the column4 classes

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job so far guys you really got my attention :clap: :clap:, I think you made extras effort and features the project should be only a teardown of the original one but you made it as it is. good job :tophat: :tophat:, but in order to make your project approved you need to make some changes:

- Make the footer content smaller to fix this issue.
image

- Validate your HTML in the [here](https://validator.w3.org/), while doing that I found a lot of errors:
image

### Please fix all the required changes before you submit another code review.
### :tophat: Happy Coding

File: index.html (hunk 2 above)
Comment (umairarshadbutt) [reply index.html]: Done

File: README.md (hunk 1 above)
Comment (umairarshadbutt) [reply README.md]: Done

**Review by elmejdki — APPROVED** [review]
# Great job guys :clap: thanks for making the required changes
# status: approved 

You can now merge to master your project was approved.

### Happy Coding

**Pull request merged**